omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -41,6 +41,9 @@
 
 default['private_chef']['fips_enabled'] = ChefConfig.fips?
 
+default['private_chef']['required_recipe']['enable'] = false
```

Comment:
I'm wondering if we want both of these?  We could alternatively check if path was non-nil.  If we do want both of these, we might want to add a check that path is set when enable is true to avoid a potentially confusing error message during reconfigure.